Add overlay/content example (#966)
reactjs/react-modal: docs/styles/transitions.md

}
```

#### Styling the overlay and content

> It is possible to independently style the transitions for the modal 
> overlay and modal content. The example below will fade the overlay 
> in and slide the modal content on-screen from the bottom of the page:

Reviewer: ```suggestion
It is possible to independently style the transitions for the modal 
overlay and modal content. 

{about precendence}

The example below will fade the overlay 
in and slide the modal content on-screen from the bottom of the page:
```

```javascript
{
  <Modal
    style={{
      overlay: { background: 'rgba(0, 0, 0, 0.3)' },
      content: {
        top: '50%',
        left: '50%',
        right: 'auto',
        bottom: 'auto',
        marginRight: '-50%',

>         // transform: 'translate(-50%, -50%)', This will override the css values if not removed

Reviewer: Maybe we can move this note after the first paragraph to warn about the precedence.

}
```

#### Styling the overlay and content

It is possible to independently style the transitions for the modal 
overlay and modal content. 

> Keep in mind that inline styles will take precedence over those defined 
> as CSS rules. If you want to apply a transition to one of the default styles,
> such as `transform`, you can remove it from the inline definition and define
> it instead as a CSS rule.

Reviewer: ```suggestion
Keep in mind that inline styles will take precedence over those defined 
as CSS rules. In case you need to use both at the same time, you'll need to use the css `!important`.
```

Reviewer: Just made more general because it will happen to other properties as well.

The example below will fade the overlay 
in and slide the modal content on-screen from the bottom of the page:

```javascript
{
  <Modal
    style={{
      overlay: { background: 'rgba(0, 0, 0, 0.3)' },
      content: {
        top: '50%',
        left: '50%',
        right: 'auto',
        bottom: 'auto',
        marginRight: '-50%',

>         // transform: 'translate(-50%, -50%)',

Reviewer: ```suggestion
        // transform: 'translate(-50%, -50%)', if you use inline styles instead of css 
```

Reviewer: This line is to clarify why this line is commented.
